### PR TITLE
COC: add code of conduct page

### LINKED ISSUE #28: Contributor Covenant CODE OF CONDUCT
We need documents related to this.

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
Update COC page.

### CODE REVIEW

File: docs/coc.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at your@email.com. All
```
Comment (RuffaloLavoisier) [inline docs/coc.md]: @uvwxy please check email.
I think I just need to fix this part.

**Comment by RuffaloLavoisier** [comment]
And make sure that the page is in the right position.

File: docs/coc.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
```
Comment (simonmicro) [inline docs/coc.md]: You may put this line into the one before. Line break for ONE word?!

File: docs/coc.md (hunk 1 above)
Comment (simonmicro) [reply docs/coc.md]: Maybe put a channel in our Discord? Se we all can react? I know there are even some ticket-bots, which we could utilize for that...

File: docs/coc.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see
+https://www.contributor-covenant.org/faq
```
Comment (simonmicro) [inline docs/coc.md:75]: Maybe we should also mention from where we copy-pasted that :P

Comment (RuffaloLavoisier) [reply docs/coc.md:75]: What is mean?

File: docs/coc.md (hunk 2 above)
Comment (RuffaloLavoisier) [reply docs/coc.md]: And where do you think it is appropriate to put the current page?

**Comment by RuffaloLavoisier** [comment]
image

**Pull request merged**